k-distribution/src/main/scripts/bin/k-configure-opam-common

```diff
@@ -1,18 +1,19 @@
 add_repo() {
-  opam repository add k "$(dirname "$0")/../lib/opam" \
-   || opam repository set-url k "$(dirname "$0")/../lib/opam"
+  opam repository add k "$(dirname "$0")/../lib/kframework/opam" \
+   || opam repository set-url k "$(dirname "$0")/../lib/kframework/opam"
   opam update
 }
 
-OCAML_VERSION="4.06.1+k$1"
 case "$(opam --version)" in
 1.*)
+        OCAML_VERSION="4.06.1+k$1"
 	echo Detected Opam version 1
 	opam init $INIT_ARGS
 	add_repo
 	opam switch "$OCAML_VERSION"
 	;;
 2.*)
+        OCAML_VERSION="4.10.0$1"
```

Comment:
Spacing of these lines looks weird.

Reply:
Mixing spaces with tabs, it seems... Should we pick one of the two (at least, one per language) and convert all sources?

Reply:
No need to convert all sources, we can just make sure we don't introduce new errors and fix ones we find along the way.